BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -1658,60 +1659,38 @@ public void PAWRefresh()
         [KSPEvent(advancedTweakable = true, guiActive = true, guiActiveEditor = true, guiName = "#LOC_BDArmory_FireAngleOverride_Enable", active = true)]//Disable fire angle override
         public void ToggleOverrideAngle()
         {
-            using (List<Part>.Enumerator craftPart = EditorLogic.fetch.ship.parts.GetEnumerator())
-                while (craftPart.MoveNext())
-                {
-                    if (craftPart.Current == null) continue;
-                    if (craftPart.Current.name != part.name) continue;
-                    using (List<ModuleWeapon>.Enumerator weapon = craftPart.Current.FindModulesImplementing<ModuleWeapon>().GetEnumerator())
-                        while (weapon.MoveNext())
-                        {
-                            if (weapon.Current == null) continue;
-                            weapon.Current.FireAngleOverride = !weapon.Current.FireAngleOverride;
-                            if (weapon.Current.FireAngleOverride == false)
-                            {
-                                weapon.Current.Events["ToggleOverrideAngle"].guiName = StringUtils.Localize("#LOC_BDArmory_FireAngleOverride_Enable");// Enable Firing Angle Override
-                            }
-                            else
-                            {
-                                weapon.Current.Events["ToggleOverrideAngle"].guiName = StringUtils.Localize("#LOC_BDArmory_FireAngleOverride_Disable");// Disable Firing Angle Override
-                            }
+            FireAngleOverride = !FireAngleOverride;
+            if (FireAngleOverride == false)
```

Comment:
`if (!FireAngleOverride)` would be cleaner (like with `BurstOverride` below), or `if (fireAngleOverride)` and switch the blocks around.